### PR TITLE
Custom logout page

### PR DESCRIPTION
**Pull request by mbertrand**
#### What are the relevant tickets?
Fixes https://github.com/mitodl/odl-video-service/issues/56

#### What's this PR do?
- Adds a custom logout HTML template and changes the order of settings.INSTALLED_APPS so that a user is not redirected to the admin logout page after logging out.
- Changes the 'login' link on the base template to redirect to the index page (if touchstone integration is on, they will logged in via that; otherwise there will be a login form on the index page).

#### How should this be manually tested?
Log in, then click the 'log out' link.  You should be redirected to a simple logout page and not the admin logout page. Click the 'Login' link.  You should be redirected to the home page with a login form.

### CODE REVIEW

**Comment by giocalitri** [comment]
Why did you choose this approach instead of using the standard django logout feature with simple redirect to the index page via the `next` get parameter (if I recall well)? Basically what we are doing in Micromasters.

File: ui/templates/registration/logged_out.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{% extends "base.html" %}
+{% block title %}Logout{% endblock %}
+{% block pagetitle %}Logout{% endblock %}
+{% block content %}
+You are now logged out.
+{% endblock %}
```
Comment (giocalitri) [inline ui/templates/registration/logged_out.html]: please put an extra blank line at the end

File: odl_video/settings.py [hunk 2]
```diff
@@ -104,6 +104,9 @@ def get_var(name, default):
 # Application definition
 
 INSTALLED_APPS = [
+    'ui.apps.UIConfig',
+    'cloudsync.apps.CloudSyncConfig',
+    'dj_elastictranscoder',
```
Comment (giocalitri) [inline odl_video/settings.py]: can you please explain me why the change of order and if you see any side effect by doing this?

Comment (mbertrand) [inline odl_video/settings.py]: `ui` in particular had to be before `django.contrib.admin` to override the admin logout page.  I decided to also place `cloudsync` and `dj_elastictranscoder` above the standard django apps to preemptively avoid similar issues.  I ran unit tests and manually tested the app to see if there were any unintended side effects but did not see any.  If you prefer I can put `cloudsync` and `dj_elastictranscoder` back in their original order.

Comment (giocalitri) [inline odl_video/settings.py]: I have no preferences, but I was wondering if you looked for possible ones.
I remember from reading documentation for unrelated issues that the sequence is pretty important, so I thought you looked into it.
Anyway I think we will find out.

File: ui/templates/registration/logged_out.html (hunk 1 above)
Comment (mbertrand) [inline ui/templates/registration/logged_out.html]: Blank line added

**Comment by giocalitri** [comment]
👍
